packages/fast-components-react-base/src/index.ts

```diff
@@ -2,6 +2,10 @@ import Button from "./button";
 export {Button};
 export * from "./button";
 
+import Toggle from "./toggle";
```

Comment:
Should alphabetize

Reply:
Like above, I think we should have our components separate from our other stuff. If we vote otherwise, I'm fine with it though.

Reply:
Foundation is technically a component. I think maintainability is key unless we are going to comment and denote the intentional difference in structure. I think alphabetizing keeps everything in easy to understand and intuitive, but we can discuss as a team.